# Solución de examen 1

## Situación 3

Antonio, un profesor de matemáticas participará en el Tour de Francia (competencia de ciclismo) donde usará su bicicleta de ruedas cuadradas. La longitud de uno de los recorridos es de $192\ \rm{km}$.

**9. Si las ruedas cuadradas tienen $50\ \rm{cm}$ de lado, ¿cuántas vueltas tendrá que dar una de las ruedas para llegar a la meta?**

$$ P=4L $$

$$ P=4(50\ \rm{cm}) $$

$$ P=200\ \rm{cm} $$

Primero tenemos que convertir $192\ \rm{km}$ en $\rm{cm}$.

$$ 192\ \rm{km}=192000\ \rm{m}=19200000\ \rm{cm} $$

$$ \text{Número de veces que gira la rueda}=\dfrac{\text{longitud de la pista}}{\text{perímetro de la rueda}} $$

$$ \text{Número de veces que gira la rueda}=\dfrac{19200000\ \rm{cm}}{200\ \rm{cm}}=96000 $$

In [1]:
19200000/200

96000.0

a) 7680

b) 76800

c) 96000

d) 192000

**Solución:** c)

**10. Marie, una ciclista francesa, también se encuentra en la competencia. Su bicicleta tiene ruedas circulares de $50\ \rm{cm}$ de diámetro. Si Marie pedalea hasta lograr que su rueda gire a $45^\circ$ cada centésima de segundo, ¿cuánto tiempo tardará en llegar a la meta?**

$$ \text{Longitud de arco}=\theta\cdot R $$

El ángulo $\theta$ tiene que estar en radianes.

Necesitamos convertir $45^\circ$ a radianes.

$$ 45^\circ=\dfrac{45\cdot\pi}{180}\ \rm{rad} $$

$$ 45^\circ=\dfrac{45\cdot\pi}{180}\ \rm{rad}=\dfrac{\pi}{4}\ \rm{rad} $$

In [2]:
45*π/180

0.7853981633974483

El ángulo es
$$ \theta=0.78539\ \rm{rad} $$

$$ \text{Longitud de arco}=(0.78539\ \rm{rad})\cdot(25\ \rm{cm})=19.63475\ \rm{cm} $$

In [4]:
19200000*(1/100)/19.63475

9778.581341753777

$$ \text{tiempo  que tardó en recorrer toda la pista}=9778.5813\ \rm{s} $$

In [5]:
9778.58134/60

162.97635566666668

In [7]:
162.9763/60

2.7162716666666666

$$ \text{tiempo  que tardó en recorrer toda la pista}=2.71\ \text{horas}$$

$$ v=\dfrac{\text{longitud de la pista}}{\text{tiempo que tardó}}=70.84\ \dfrac{\rm{km}}{\rm{hora}}\ $$

In [8]:
192/2.71

70.84870848708488

a) Menos de 2 horas

b) Más de 2 horas pero menos 4 horas

c) Más de 4 horas pero menos 6 horas

d) Ninguna de las anteriores

**Solución:** b)

**11. Marie cambia las ruedas por otras que tienen un rin de $55\ \rm{cm}$ de diámetro con una llanta de $3\ \rm{cm}$ de ancho. ¿Cuánta distancia recorrerá la nueva rueda al dar una vuelta?**

<img src="rines.png" align="middle" width="400">

$$ P=\pi\cdot D=2\pi R $$

El nuevo radio es $30.5\ \rm{cm}$

a) $122\pi\ \rm{cm}$

b) $116\pi\ \rm{cm}$

c) $61\pi\ \rm{cm}$

d) $58\pi\ \rm{cm}$

**Solución:** c)

**12. Marie quiere pintar los rines de sus nuevas ruedas de color naranja. Cada rayo (sector circular) tiene un ángulo de $30^\circ$ desde el centro de la rueda. Si la pintura cuesta $0.5$ euros el mililitro y un mililitro de pintura rinde para $130 \rm{cm}^2$, ¿cuánto gastará para pintar ambas cara de sus dos rines?**

$$ A=\dfrac{\theta\cdot r^2}{2} $$

El ángulo $\theta$ lo sustuimos en radianes.

$$ 30^\circ=\dfrac{30\cdot\pi}{180}\ \rm{rad}=\dfrac{\pi}{6}\ \rm{rad}=0.523\ \rm{rad}$$

In [9]:
30*π/180

0.5235987755982988

$$ A=\dfrac{(0.523\ \rm{rad})\cdot (27.5\ \rm{cm})^2}{2} $$

In [10]:
0.523*(27.5)^2/2

197.759375

$$ A=197.75\ \rm{cm}^2 $$

Necesitamos pintar $16$ áreas iguales.

In [11]:
197.759375*16

3164.15

$$ \text{Área total a pintar}=3164.15\ \rm{cm}^2 $$

In [12]:
3164.15/130

24.339615384615385

Necesito $24.33\ \rm{mL}$ para pintar las ruedas.

In [13]:
24.339615384615385*0.5

12.169807692307693

La chamba salió en $12.16$ euros

<img src="rines.png" align="middle" width="400">

a) Menos de $5$ euros

b) Más de $5$ euros pero menor de $10$ euros

c) Más de $10$ euros pero menor de $15$ euros

d) Más de $15$ euros

**Solución:** c)

## Situación 1

**1. La diagonal de un cuadrado mide $x$ unidades. Su área, en unidades cuadradas, es**

<img src="fractal1.png" align="middle" width="160">

Teorema de Pitágoras:

$$ L^2+L^2=x^2 $$

$$ 2L^2=x^2 $$

Despejamos $L$:

$$ L^2=\dfrac{x^2}{2} $$

$$ L=\sqrt{\dfrac{x^2}{2}} $$

El área de un cuadrado es

$$ A=L\cdot L=L^2=\dfrac{x^2}{2} $$

a) $\dfrac{x}{\sqrt{2}}$

b) $\sqrt{2}x^2$

c) $\dfrac{x^2}{2}$

d) $\dfrac{\sqrt{3}}{4}x$

**Solución:** c)

**2. Para la figura que se muestra, la proporción del área del triángulo con respeto al área del cuadrado es**

<img src="fractal2.png" align="middle" width="175">

a) $\dfrac{1}{4}$

b) $\dfrac{1}{8}$

c) $\dfrac{1}{9}$

d) $\dfrac{1}{16}$

**Solución:** 

**3. El área sombreada de la siguiente figura, en unidades cuadradas, es**

<img src="fractal3.png" align="middle" width="300">

a) $2x^2+\dfrac{1}{4}$

b) $\dfrac{x^2}{2}+\dfrac{1}{8}$

c) $\dfrac{7}{16}x^2$

d) $\dfrac{3}{4}x^2$

**Solución:** 

**4. El área sombreada de la siguiente figura, en unidades cuadradas, es**

<img src="fractal4.png" align="middle" width="300">

a) $\dfrac{3}{2}x^2$

b) $\dfrac{5}{4}x^2$

c) $\dfrac{7}{8}x^2$

d) $\dfrac{3}{4}x^2$

**Solución:** 

# Álgebra para analizar objetos geométricos

# Clase 1

### Ejes coordenados

Eje $x$ (eje de las abscisas)

Eje $y$ (eje de las ordenadas)

**¿Cómo nos movemos en el plano cartesiano?**

Para empezar a movernos en el plano cartesiano tenemos que partir del origen $O(0,0)$

Si las coordenadas son $(x,y)$, la $x$ nos indica cuántas unidades movernos sobre el eje de las $x$ (si es positivo nos movemos hacia la derecha; si es negativo, hacia la izquierda).

La segunda coordenada (la ordenada $y$) nos indica cuántas unidades movernos sobre el eje de las $y$ (si es positivo nos movemos hacia arriba; si es negativo, hacia abajo).

$P_1(1,3)$

$P_2(-2,2)$

$P_3(-4,-3)$

$P_4(2,-4)$

Cuadrantes:
* Primer cuadrante $(+,+)$
* Segundo cuadrante $(-,+)$
* Tercer cuadrante $(-,-)$
* Cuarto cuadrante $(+,-)$


## Distancia

Si tenemos dos puntos $P(x_1,y_1)$ y $Q(x_2,y_2)$, la longitud del segmento entre estos puntos es

$$ \overline{PQ}=\sqrt{(x_1-x_2)^2+(y_1-y_2)^2} $$

* ***Ejemplo:*** Determine la distancia entre los puntos $A(-2,1)$, y $B(1,5)$

$$ \overline{AB}=\sqrt{(x_1-x_2)^2+(y_1-y_2)^2} $$

$$ \overline{AB}=\sqrt{[(-2)-(1)]^2+[(1)-(5)]^2} $$

$$ =\sqrt{[-2-1]^2+[1-5]^2} $$

$$ =\sqrt{[-3]^2+[-4]^2} $$

$$ =\sqrt{9+16} $$

$$ =\sqrt{25} $$

$$ =5 $$

* ***Ejemplo:*** Determine el punto equidistante entre los puntos $A(-2,3)$, $B(4,1)$, y $C(1,7)$

Pensaremos que el cuarto punto que sea equidistante a los demás es $P(x,y)$:

$$ \overline{AP}=\overline{BP}=\overline{CP} $$

¿Cómo es el segmento $\overline{AP}$?

$$ \overline{AP}=\sqrt{[(-2)-(x)]^2+[(3)-(y)]^2} $$

$$ \overline{AP}=\sqrt{(-2-x)^2+(3-y)^2} $$

¿Cómo es el segmento $\overline{BP}$?

$$ \overline{BP}=\sqrt{[(4)-(x)]^2+[(1)-(y)]^2} $$

$$ \overline{BP}=\sqrt{(4-x)^2+(1-y)^2} $$

¿Cómo es el segmento $\overline{CP}$?

$$ \overline{CP}=\sqrt{[(1)-(x)]^2+[(7)-(y)]^2} $$

$$ \overline{CP}=\sqrt{(1-x)^2+(7-y)^2} $$

**1ra ecuación**

$$ \overline{AP}=\overline{BP} $$

$$ \sqrt{(-2-x)^2+(3-y)^2}\quad=\quad\sqrt{(4-x)^2+(1-y)^2} $$

Elevamos el cuadrado:

$$ (-2-x)^2+(3-y)^2=(4-x)^2+(1-y)^2 $$

Desarrollamos los binomios

$$ (-2-x)^2=(-2)^2+2(-2)(-x)+(-x)^2=4+4x+x^2 $$

$$ (3-y)^2=(3)^2+2(3)(-y)+(-y)^2=9-6y+y^2 $$

*"El binomio al cuadrado es el primer término al cuadrado más el doble del primer término por el segundo término más el segundo término al cuadrado"*

$$ (4-x)^2=(4)^2+2(4)(-x)+(-x)^2=16-8x+x^2 $$

$$ (1-y)^2=(1)^2+2(1)(-y)+(-y)^2=1-2y+y^2 $$

Sustituimos

$$ (4+4x+x^2)+(9-6y+y^2)=(16-8x+x^2)+(1-2y+y^2) $$

$$ 4x+x^2-6y+y^2+8x-x^2+2y-y^2=16+1-4-9 $$

$$ 12x-4y=4 $$

**2da ecuación**

$$ \overline{BP}=\overline{CP} $$

**Tarea:** Terminen el ejercicio y encuentren que el punto es $P(3/2,7/2)$.

* ***Ejemplo:*** Determina si el triángulo formado por los puntos $A(0,0)$, $B(6,5)$, y $C(1,6)$ es escaleno, isósceles o equilátero.